packages/build/src/steps/run_core_steps.js

```diff
@@ -186,9 +191,10 @@ const runBuildStep = async function ({
     debug,
     timers: [],
     featureFlags,
+    // eslint-disable-next-line max-lines
     childEnv,
     repositoryRoot,
+    systemLog,
   })
-
```

Comment:
I suspect you've removed this because of the `max-lines` rule? 😄

Reply:
Yeah 😓